components/board/pi/impl/board.go

```diff
@@ -73,16 +76,21 @@ type piPigpio struct {
 	analogs         map[string]board.AnalogReader
 	i2cs            map[string]board.I2C
 	spis            map[string]board.SPI
-	interrupts      map[string]board.DigitalInterrupt // Maps interrupt names to interrupts
-	interruptsHW    map[uint]board.DigitalInterrupt   // Maps broadcom addresses to the same
-	logger          golog.Logger
-	isClosed        bool
+	// `interrupts` maps interrupt names to the interrupts. `interruptsHW` maps broadcom addresses
+	// to these same values.
+	interrupts   map[string]board.ReconfigurableDigitalInterrupt
+	interruptsHW map[uint]board.ReconfigurableDigitalInterrupt
```

Comment:
[nit] moving these to be closed to the other interrupt states would be cool

Reply:
There is a misnamed context and its cancel function, which (I suspect) shouldn't be used solely with the interrupts. I'll rename them.